dashboard/test/fixtures/schools.tsv

```diff
@@ -0,0 +1,17 @@
+NCESSCH	LEAID	SCHNAM	CHARTR
+010000200277	0100002	SEQUOYAH SCH - CHALKVILLE CAMPUS	N
+010000201402	0100002	EUFAULA SCH - EUFAULA CAMPUS	N
+010000500870	0100005	ALA AVENUE MIDDLE SCH	N
+010000500871	0100005	ALBERTVILLE HIGH SCH	N
+020000100206	0200001	JOANN A. ALEXIE MEMORIAL SCHOOL	2
+020000100207	0200001	GLADYS JUNG ELEMENTARY	2
+020000300216	0200003	ALAKANUK SCHOOL	2
+020000300217	0200003	EMMONAK SCHOOL	2
+060000113121	0600001	ASSURANCE LEARNING ACADEMY	1
+060000113717	0600001	ALBERT EINSTEIN ACADEMY ELEMENTARY	1
+060001411118	0600014	SUMMIT LEADERSHIP ACADEMY-HIGH DESERT	1
+060001411746	0600014	PATHWAYS TO COLLEGE	1
+480000407173	4800004	POR VIDA ACADEMY CHARTER H S	1
+480000409116	4800004	CORPUS CHRISTI COLLEGE PREP H S	1
+480001407174	4800014	GEORGE GERVIN ACADEMY	1
+480001410415	4800014	THE EDUCATION AND TRAINING CENTER	1
```

Comment:
OK, it seems like we should agree on when to use the subset of the data before considering whether to change what's in the subset. If we decide to use the subset in more places, then I agree it might become "material" to change what's in the subset.

Using the subset of schools in staging and development would actually be kind of tricky to do. If we use the subset of schools, then we probably want to use the subset of school districts, since that guarantees that any district the developer selects in a dropdown will actually have some schools in it. If we don't do this, then the feature will appear broken because for 99% of districts that you can choose from, there will appear to be no schools in it. This will be especially undesirable once the district and school dropdowns are on the teacher sign-up page which developers use frequently.

The complication is that development and staging already have the full set of `school_districts`, which are depended on as a foreign key by `school_infos`, which are presumably depended on by enrollments and possibly other things. So rails won't let us just delete all the `school_districts` data because it will be a foreign key violation. 

So, based on how complicated it will get to reduce `school_districts` on development and test down to the subset, I really don't think it's worth the time just to avoid a 4-minute seed one time for staging and existing developers. It'll be a little annoying, but I feel less bad knowing that within a day staging will come out "ahead" since each subsequent seed will be taking 30 seconds less than before due to skipping the re-seed of the school districts.

Ideally though, contributors and new employees would not have to pay this cost when they create a new database, while existing developers (and staging) would use the complete data (to match what they already have in `school_districts`). As a proxy for this, what do you think about leaving `stub_school_data` as `false` in development for now, and in a month or two once most existing developers have run `rake seed:all`, we can change it to default to `true`?